packages/hardhat/contracts/periphery/lib/ControllerHelperDataType.sol

```diff
@@ -82,7 +82,7 @@ library ControllerHelperDataType {
         bool burnExactRemoved;
     }
 
-    /// @dev params for batchMintLp()
+    /// @dev params for wMintLp()
```

Comment:
wMint?

Reply:
fair enough

Reply:
Yeah felt like `mintLp()` looks like minting an LP position, where `wMint` could mean more minting Wtoken.
We are using the same in Controller.sol for wMint...etc